web/app/[locale]/pricing/page.tsx

```
@@ -0,0 +1,403 @@
+                    name={t("team.name")}
+                    price={`${t("team.price")}${t("perUserMonth")}`}
+                  />
+                  <ColumnHead
+                    name={t("enterprise.name")}
+                    price={t("enterprise.price")}
+                  />
+                </tr>
+              </thead>
+              <tbody>
+                {compareRows.map((row, i) => (
+                  <tr key={i} className="border-b border-border">
```

Biome (2.5.0)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)